README.md

```diff
@@ -25,7 +25,53 @@ The following tool(s) must be installed on your system prior to developing the w
 We also assume you've got GNU coreutils installed, which is usually the case by default on Linux
 and should be installable via Homebrew on macOS.
 
-## Website Development
+## Website Development Guides
+
+### Where's the documentation content?
+
+All docs go under `content/`; most changes are usually made in `content/docs` which is conceptually
+the "latest" version of the docs and includes other sections which don't make sense to version. For
```

Comment:
Question: is it not the case that currently for the stuff for which it _does_ make sense to be versioned /content/docs should _only_ include documentation relevant for latest released cert-manager version- should this be explicitly stated here (perhaps as well as the fact that documentation aimed at a next release should be PR-ed against next-docs?
(This has been discussed multiple times, however it's late and my brain fails)

Reply:
Fair comment yeah - I think this might look better with less words, I'll take another crack at it